src/System.Security.Cryptography.OpenSsl/src/Resources/Strings.resx

```diff
@@ -132,6 +132,12 @@
   <data name="Cryptography_HashAlgorithmNameNullOrEmpty" xml:space="preserve">
     <value>The hash algorithm name cannot be null or empty.</value>
   </data>
+  <data name="Cryptography_CurveNotSupported" xml:space="preserve">
+    <value>The specified curve '{0}' or its parameters are not valid for this platform.</value>
+  </data>
+  <data name="Cryptography_InvalidKeySize" xml:space="preserve">
```

Comment:
Actually I think the text for the other Cryptograph_InvalidKeySizes (including CNG) is fine. Unless you object, I will change this openssl string to: "Specified key is not a valid size for this algorithm." to match the other usages